.gitignore

```diff
@@ -61,3 +61,4 @@ typings/
 .DS_Store
 
 dist/
+tmp/
```

Comment:
Currently, karma uses `./coverage` folder to store the coverage results.
I think that we could be able to reuse that folder and consolidate the results in there.

IMO there shouldn't be collisions in folders as `karma` uses browser's name-version pair as folder to store the results. We on the other side could use `./coverage/qunit`, for example.
We'd need to enhance a bit the configuration a bit, so it'd become like this:

`ui5.yaml`
```yaml
...
server:
  customMiddleware:
    - name: "@ui5/middleware-code-coverage"
      afterMiddleware: compression
      configuration:
            report:
              "report-dir": "./coverage/qunit"
```

Reply:
@matz3 whats your opinion on that one?